docs/user/exercises/test-case-best-practices.inc

```diff
@@ -0,0 +1,406 @@
+General best practices
+^^^^^^^^^^^^^^^^^^^^^^
+
+- **Write meaningful comments for your tests**
+
+  These comments should contain information about what is tested specifically, which task from the problem statement
+  is addressed or which TODO (if there are numbered TODOs in the template), how many points this test is worth when
+  passed and more if necessary.
+
+  .. code-block:: java
+
+    /**
+     * Tests that borrow() in Book successfully sets the available attribute to false
+     * Problem Statement Task 2.1
+     * Worth 1.5 Points (Weight: 1)
```

Comment:
As long as the exercise is only graded automatically, I'd leave the last line out. This will probably create some inconsistencies with the actual grading entered on Artemis when something is changed